test-network/scripts/deployCC.sh

```diff
@@ -42,6 +42,9 @@ if [ "$CC_SRC_PATH" = "NA" ]; then
   if [ "$CC_NAME" = "basic" ]; then
     println $'\e[0;32m'asset-transfer-basic$'\e[0m' chaincode
     CC_SRC_PATH="../asset-transfer-basic"
+  elif [ "$CC_NAME" = "events" ]; then
+    println $'\e[0;32m'asset-transfer-events$'\e[0m' chaincode
+    CC_SRC_PATH="../asset-transfer-events"
```

Comment:
I'm not fond on keeping changing test network to support the different samples we have. I'd rather we keep it to a minimum and use the override mechanism that's in place to specify the chaincode through arguments.

Reply:
We can probably keep the 5 we have as the main asset transfer series, and move this as another sample. Events are not really related to the APIs you would use for transfers.